helm/prometheus-rules/templates/platform/atlas/alerting-rules/loki.rules.yml

```diff
@@ -227,7 +227,7 @@ spec:
     - alert: LokiLogTenantIdMissing
       annotations:
         description: '{{`Log lines are being dropped due to missing tenant id.`}}'
-        runbook_url: loki/
+        runbook_url: loki-log/
```

Comment:
Let's wait till i've merged my prs tomorrow but you'll have to add the full link

Reply:
updated to full link :heavy_check_mark: